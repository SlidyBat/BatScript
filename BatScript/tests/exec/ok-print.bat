print 1
print 2.0
print "Hello"
print true
print false