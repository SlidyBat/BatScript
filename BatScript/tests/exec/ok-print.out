1
2
Hello
1
0
